bundles/framework/myplacesimport/StyleForm.js

```diff
@@ -58,10 +58,8 @@ Oskari.clazz.define(
          * @param {Object} style
          */
         setStyleValues: function (style) {
-            style.dot.color = this._formatColorFromServer(style.dot.color);
-            style.line.color = this._formatColorFromServer(style.line.color);
-            style.area.lineColor = (typeof style.area.lineColor === 'string' ? this._formatColorFromServer(style.area.lineColor) : null);
-            style.area.fillColor = (typeof style.area.fillColor === 'string' ? this._formatColorFromServer(style.area.fillColor) : null);
+            style.area.lineColor = (typeof style.area.lineColor === 'string' ? style.area.lineColor : null);
```

Comment:
Can _prefixColorForServer() and _formatColorFromServer() be removed now that the calls have been removed?

Reply:
Yep. Added new commit.